.golangci.yaml

```diff
@@ -0,0 +1,142 @@
+## golangci-lint v1.55.2
+
+# References:
+# - https://golangci-lint.run/usage/linters/
+# - https://gist.github.com/maratori/47a4d00457a92aa426dbd48a18776322
+
+run:
+  timeout: 10m # default 1m
+
+linters-settings:
+  gosimple:
+    go: "1.21" # default 1.13
+  govet:
+    enable-all: true
+    disable:
+      - fieldalignment # too strict
+      - shadow # too strict
+  staticcheck:
+    go: "1.21" # default 1.13
+
+  # Non-default
+  cyclop:
+    # Top 3:
```

Comment:
Can you please try reverting cyclop.max-complexity and gocognit.min-complexity to their defaults and recalibrate accordingly? These values are not applicable to this code base.